kratos/python/add_model_to_python.cpp

```diff
@@ -40,7 +56,22 @@ void  AddModelToPython(pybind11::module& m)
         .def("HasModelPart", &Model::HasModelPart)
         .def("GetModelPartNames", &Model::GetModelPartNames)
         .def("__getitem__", &Model_GetModelPart, py::return_value_policy::reference_internal)
-        .def("__str__", PrintObject<Model>);
+        .def("__str__", PrintObject<Model>)
+        ;
+    VariableIndexingUtility< ModelBinder, Variable<bool> >(model_binder);
```

Comment:
all of this is not needed as you will use the interface of the DataValueContainer